layers-samples/eqrt-video.html

```diff
@@ -0,0 +1,268 @@
+  <!-- Stereo layer dependency -->
+  <script src="../js/wglu/wglu-program.js"></script>
+  <script src="../js/wglu/wglu-url.js"></script>
+  <script src="../js/stereo-util.js"></script>
+</head>
+
+<body>
+  <header style="max-width: 800px;">
+    <details open>
+      <summary>Equirect Video</summary>
+      <p>
+        This sample shows how to show 180/360 video via an equirect layer, with 
+        the addition of media control bar that enable fast forward/backward, 
+        pause/play and exit. Equirect layer is similar in terms of its advantage 
+        over projection. It can be used for rendering static background like cube 
+        layer, like demonstrated in the Equirect Layer example, but the real edge 
+        of using equirect layer over cube layer is to show immersive video.
+        <a class="back" href="./index.html">Back</a>
+      </p>
+      <label>Choose a video</label>
+      <select id="eqrtVideoSelect">
+        <option value="https://d25a56pc18k0co.cloudfront.net/sloths_binaural_3840_180_3D-injected.mp4"
+          data-video-width=3840 data-video-height=1920 data-video-layout="stereo-left-right" data-video-angle=180>
+          stereo-left-right 180 3840x1920 30fps
+        </option>
+      </select><br />
+      <label>Video rendering FPS</label><input type="number" id="eqrtVideoFPS" value="60" /><br />
+    </details>
+  </header>
+  <main style='text-align: center;'>
+    <p>Click 'Enter VR' to see content</p>
+  </main>
+  <script type="module">
+    import { WebXRButton } from '../js/util/webxr-button.js';
+    import { Scene, WebXRView } from '../js/render/scenes/scene.js';
+    import { Renderer, createWebGLContext } from '../js/render/core/renderer.js';
+    import { QueryArgs } from '../js/util/query-args.js';
+    import { MenuSystem } from '../js/render/nodes/menu-system.js';
+
+    const EQRT_180_VIDEO_STEREO_LR_PATH = 'https://d25a56pc18k0co.cloudfront.net/sloths_binaural_3840_180_3D-injected.mp4';
+
+    // If requested, use the polyfill to provide support for mobile devices
+    // and devices which only support WebVR.
+    import WebXRPolyfill from '../js/third-party/webxr-polyfill/build/webxr-polyfill.module.js';
+    if (QueryArgs.getBool('usePolyfill', true)) {
+      let polyfill = new WebXRPolyfill();
+    }
+
+    // XR globals.
+    let xrButton = null;
+    let xrSession = null;
+    let xrRefSpace = null;
+    let xrGLFactory = null;
+    let xrFramebuffer = null;
+    let stereoUtil = null;
+
+    // WebGL scene globals.
+    let gl = null;
+    let renderer = null;
+    let scene = new Scene();
+
+    // Layer globals
+    let projLayer = null;
+    let eqrtLayer = null;
+    let eqrtVideoElement = null;
+    let eqrtVideoWidth = 0;
+    let eqrtVideoHeight = 0;
+    let eqrtVideoNeedsUpdate = false;
+
+    // Set up media control bar
+    let menuSystem = new MenuSystem();
+    menuSystem.createButton('../media/textures/backward-button.png', () => { eqrtVideoElement.currentTime -= 15; });
```

Comment:
The forward and backwards buttons introduced unfortunately long pauses for me, and at first I thought they were broken. Might be a side effect of how I'm serving the assets locally. Nothing you need to change, just figured I'd mention that it was a point of confusion for me initially.

Reply:
This happens to me too sometimes, I added a note in the description.